wandb/sdk/service/server.py

```diff
@@ -126,9 +128,13 @@ def serve(self) -> None:
         self._inform_used_ports(grpc_port=grpc_port, sock_port=sock_port)
         setproctitle = wandb.util.get_optional_module("setproctitle")
         if setproctitle:
-            service_ver = 0
+            service_ver = 2
```

Comment:
should this one be some global variable?

Reply:
Sort of... covered in the comment a couple lines lower:
```
            # this format is similar to wandb_manager token but it purely informative now
            # (consider unifying this in the future)
```
They are serving slightly different purposes right now -- but we might want them to be the same eventually.   Right now it is just for debugging (and only if they installed with `pip install wandb[service]` btw)

Eventually it should probably end up using the `_ManagerToken` class and be unified.  we might want to add some sort of auth token in there also, thats why i didnt unify it yet.